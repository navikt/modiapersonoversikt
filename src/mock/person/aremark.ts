import { Diskresjonskoder, TilrettelagtKommunikasjonsTyper } from '../../konstanter';
import { Kjønn, Person, Relasjonstype, SivilstandTyper } from '../../models/person/person';

export const aremark: Person = {
    fødselsnummer: '10108000398',
    kjønn: Kjønn.Mann,
    geografiskTilknytning: '0118',
    alder: 42,
    navn: {
        sammensatt: 'AREMARK TESTFAMILIEN',
        fornavn: 'AREMARK',
        mellomnavn: '',
        etternavn: 'TESTFAMILIEN',
    },
    tilrettelagtKomunikasjonsListe: [
        {
            behovKode: 'TOHJ',
            beskrivelse: TilrettelagtKommunikasjonsTyper.TOHJ
        }
    ],
    diskresjonskode: Diskresjonskoder.FORTROLIG_ADRESSE,
    statsborgerskap: 'NORSK',
    personstatus: {
        dødsdato: undefined,
        bostatus: undefined
    },
    sivilstand: {
        value: SivilstandTyper.Gift,
        beskrivelse: 'Gift',
        fraOgMed: '2005-12-12'
    },
    familierelasjoner: [{
        harSammeBosted: false,
        rolle: Relasjonstype.Barn,
        tilPerson: {
            navn: {
                sammensatt: '',
                fornavn: 'Siri',
                mellomnavn: null,
                etternavn: 'Aremark'
            },
            alder: 17,
            fødselsnummer: '10117100000',
            personstatus: {}
        }

    }],
    kontaktinformasjon: {
        mobil: {
            sistEndret: '2014-06-21T18:44:39+02:00',
            telefonnummer: '99887766',
            retningsnummer: '47',
            sistEndretAv: 'BRUKER'
        }
    }
};