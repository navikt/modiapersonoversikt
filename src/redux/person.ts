import { Person } from '../models/person';
import { Action, Dispatch } from 'redux';
import { FetchError, FetchSuccess, RestActions, STATUS } from './utils';
import { getPerson } from '../api/person-api';
import { aremark } from '../mock/person-mock';

export interface PersonState {
    data: Person;
    status: STATUS;
    error: string | null;
}

const reducerName = 'person/';

const actionNames = {
    PENDING: reducerName + STATUS.PENDING,
    OK: reducerName + STATUS.OK,
    ERROR: reducerName + STATUS.ERROR
};

const initialState: PersonState = {
    data: aremark,
    status: STATUS.NOT_STARTED,
    error: null
};

export function hentPerson(fodselsnummer: string) {
    return (dispatch: Dispatch<Action>) => {
        dispatch({ type: actionNames.PENDING});
        return getPerson(fodselsnummer)
            .then((person) => {
                dispatch({type: actionNames.OK, data: person});
                return person;
            })
            .catch((err) => {
                dispatch({type: actionNames.ERROR, error: err});
            });
    };
}

export default function reducer(state: PersonState = initialState, action: RestActions<Person>): PersonState {
    switch (action.type) {
        case actionNames.PENDING:
            return {...state, status: STATUS.PENDING };
        case actionNames.OK:
            return {...state, status: STATUS.OK, data: (<FetchSuccess<Person>> action).data};
        case actionNames.ERROR:
            return {...state, status: STATUS.ERROR, error: (<FetchError> action).error};
        default:
            return state;
    }
}