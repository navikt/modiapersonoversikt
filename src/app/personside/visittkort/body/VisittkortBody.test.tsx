import * as React from 'react';
import * as renderer from 'react-test-renderer';
import VisittkortBody from './VisittkortBody';
import { aremark } from '../../../../mock/person/aremark';
import TestProvider from '../../../../test/Testprovider';

test('viser info om bruker i visittkortbody', () => {
    const visittkortbody = renderer.create(
        <TestProvider>
            <VisittkortBody person={aremark} />
        </TestProvider>
    );

    expect(visittkortbody.toJSON()).toMatchSnapshot();
});
