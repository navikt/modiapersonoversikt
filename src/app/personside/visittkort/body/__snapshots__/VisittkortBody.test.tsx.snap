// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`viser info om bruker i visittkortbody 1`] = `
.c2 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 {
  position: absolute;
  left: -3.125rem;
  width: 3.125rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c5 > svg {
  height: 1.5rem;
  width: auto;
}

.c0 {
  background-color: white;
  padding: 1.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-top: 0.2rem;
}

.c0:focus {
  outline: none;
  box-shadow: 0 0 0 0.1875rem #254b6d;
}

.c0 > * {
  -webkit-flex: 1 1 50%;
  -ms-flex: 1 1 50%;
  flex: 1 1 50%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  margin-left: 3.125rem;
  margin-right: 3.125rem;
}

.c1:last-child {
  margin-right: 1.25rem;
}

.c1 > *:not(:last-child) {
  margin-bottom: 2.5rem;
}

.c3 {
  margin-bottom: 1rem;
}

.c4 {
  margin: 10px 0 20px 0;
}

.c4:last-child {
  margin: 10px 0 0 0;
}

.c6 {
  color: #645f5a;
  margin-top: 3px;
}

.c6 .typo-etikett-liten {
  line-height: 1rem;
}

.c10 {
  margin-bottom: 5px;
}

.c7 span:not(:last-child):after {
  content: '.';
}

.c9 {
  text-transform: capitalize;
}

.c8 {
  margin: initial;
  padding: initial;
  list-style: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c8 dt {
  -webkit-flex: 1 1 50%;
  -ms-flex: 1 1 50%;
  flex: 1 1 50%;
}

.c8 dd {
  -webkit-flex: 2 2 50%;
  -ms-flex: 2 2 50%;
  flex: 2 2 50%;
  margin: 0;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<section
  aria-label="Visittkortdetaljer"
  className="c0"
  role="region"
>
  <h2
    className="visually-hidden"
    tabIndex={-1}
  >
    Visittkortdetaljer
  </h2>
  <div
    className="c1"
  >
    <section>
      <div
        className="c2"
      >
        
        <h3
          className="typo-undertittel"
        >
          Kontaktinformasjon
        </h3>
      </div>
      <div
        className="c3"
      />
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              height={24}
              viewBox="0 0 24 24"
              width={24}
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="#C6C2BF"
              >
                <defs>
                  <mask
                    id="locationpinmask"
                  >
                    <rect
                      fill="#fff"
                      height="100%"
                      width="100%"
                    />
                    <circle
                      cx={12}
                      cy={8}
                      fill="#000"
                      r={4}
                    />
                  </mask>
                </defs>
                <path
                  d="M19.5 8c0 4.1-7.5 15.5-7.5 15.5S4.5 12.1 4.5 8a7.5 7.5 0 1 1 15 0z"
                  mask="url(#locationpinmask)"
                />
              </g>
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Bostedsadresse
          </h4>
        </div>
        <div>
          <p
            className="typo-normal"
          >
            Islandsgate 49
          </p>
          <p
            className="typo-normal"
          >
            7693 Hovedøya
          </p>
        </div>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Endret 
            25.11.2017
             
            i Folkeregisteret
          </p>
        </div>
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              height={24}
              viewBox="0 0 24 24"
              width={24}
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="#C6C2BF"
              >
                <defs>
                  <mask
                    id="locationpinmask"
                  >
                    <rect
                      fill="#fff"
                      height="100%"
                      width="100%"
                    />
                    <circle
                      cx={12}
                      cy={8}
                      fill="#000"
                      r={4}
                    />
                  </mask>
                </defs>
                <path
                  d="M19.5 8c0 4.1-7.5 15.5-7.5 15.5S4.5 12.1 4.5 8a7.5 7.5 0 1 1 15 0z"
                  mask="url(#locationpinmask)"
                />
              </g>
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Midlertidig adresse
          </h4>
        </div>
        <div>
          <p
            className="typo-normal"
          >
            Postboksnummer 9698
          </p>
          <p
            className="typo-normal"
          >
            7705 Færder fyr
          </p>
        </div>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Endret 
            10.04.2018
             
            i Folkeregisteret
          </p>
        </div>
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              height={24}
              viewBox="0 0 24 24"
              width={24}
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23.5 18c0 .8-.7 1.5-1.5 1.5H2c-.8 0-1.5-.7-1.5-1.5V6c0-.8.7-1.5 1.5-1.5h20c.8 0 1.5.7 1.5 1.5v12zm-3-9.5L12 14 3.5 8.5"
                fill="#C6C2BF"
                stroke="white"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeMiterlimit={10}
                strokeWidth="1.4"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            E-post
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          aremark@testfamilien.no
        </p>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Endret 
            15.05.2014
             i Kontakt-og reservasjonsregisteret
          </p>
        </div>
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              height={24}
              viewBox="0 0 24 24"
              width={24}
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8.6 7.9c.9-.9.9-2.4 0-3.3L6 2.2c-.9-1-2.3-1-3.2 0L1.5 3.5A3.5 3.5 0 0 0 1.1 8c3.9 5.9 9.1 11.1 15 15 1.4 1 3.2.8 4.4-.4l1.3-1.4c1-.9 1-2.3 0-3.2l-2.4-2.5c-1-.9-2.4-.9-3.3 0l-.8.8a54.9 54.9 0 0 1-7.5-7.5l.8-.8z"
                fill="#C6C2BF"
                stroke="none"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Telefon
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          112 23 344
        </p>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Endret 
            15.05.2014
             i Kontakt- og reservasjonsregisteret
          </p>
        </div>
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          
          <h4
            className="typo-element"
          >
            Telefon til bruk for NAV
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          +47 998 87 766
           (
          Mobil
          )
        </p>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Endret 
            21.06.2014
             
            av bruker
          </p>
        </div>
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              viewBox="0 0 22 22"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                transform="translate(0 8.3)"
              >
                <path
                  d="M7.3 3.7h1.2c2-.9 5-1 6.2-1v-2c-1.7.9-5.1 1.1-7.4 1.1-2.2 0-5.7-.2-7.3-1.1v1.6c0 .5 2.8 1.4 7.3 1.4"
                  fill="#C6C2BF"
                />
              </g>
              <g
                transform="translate(0 2.8)"
              >
                <path
                  d="M14.7 2.3V.7c-1.7.9-5.1 1.1-7.4 1.1-2.2 0-5.7-.2-7.3-1.1v1.6c0 .5 2.8 1.4 7.3 1.4 4.6 0 7.3-1 7.4-1.4z"
                  fill="#C6C2BF"
                />
              </g>
              <g
                transform="translate(0 12)"
              >
                <path
                  d="M0 1.5c0 .4 2.5 1.1 6.7 1.3v-.5c0-.5.3-.9.6-1.2C5 1 1.6.8 0 0v1.5z"
                  fill="#C6C2BF"
                />
              </g>
              <path
                d="M7.3 3.7c4.6 0 7.3-1 7.4-1.4C14.7.7 10.9 0 7.3 0 3.8 0 0 .7 0 2.3c0 .5 2.8 1.4 7.3 1.4"
                fill="#C6C2BF"
              />
              <g
                transform="translate(0 5.5)"
              >
                <path
                  d="M7.3 3.7c4.6 0 7.3-1 7.4-1.4V.7c-1.7.9-5.1 1.1-7.4 1.1-2.2 0-5.7-.2-7.3-1.1v1.6c0 .5 2.8 1.4 7.3 1.4"
                  fill="#C6C2BF"
                />
              </g>
              <g
                transform="translate(0 13.8)"
              >
                <path
                  d="M0 2.3c0 1.4 3.2 2.2 6.4 2.3V1.8c-2.2 0-5-.3-6.4-1.1v1.6z"
                  fill="#C6C2BF"
                />
              </g>
              <path
                d="M7.3 14.2c.2.5 3.2 1.4 7.4 1.4 4.1 0 7.1-1 7.3-1.4 0-2-5.3-2.3-7.3-2.3s-7.4.3-7.4 2.3M7.3 18v1.7c0 2 5.4 2.3 7.4 2.3s7.3-.3 7.3-2.3v-1.6c-1.8 1-5.7 1.1-7.3 1.1-1.7 0-5.6-.2-7.4-1.1"
                fill="#C6C2BF"
              />
              <path
                d="M14.7 16.5c-1.7 0-5.6-.2-7.4-1.2V17c.2.4 3.2 1.3 7.4 1.3 4.1 0 7.1-.9 7.3-1.4v-1.6c-1.8 1-5.7 1.2-7.3 1.2"
                fill="#C6C2BF"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Kontonummer
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          <span
            className="c7"
          >
            <span>
              1234
            </span>
            <span>
              56
            </span>
            <span>
              78911
            </span>
          </span>
        </p>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Endret 
            12.12.2005
             
            z9910
          </p>
        </div>
      </div>
    </section>
    <section>
      <div
        className="c2"
      >
        
        <h3
          className="typo-undertittel"
        >
          Familie
        </h3>
      </div>
      <div
        className="c3"
      />
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              height={24}
              viewBox="0 0 24 24"
              width={24}
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M12 22.4s10.5-8.1 10.5-15.8S13.2-1.7 12 6C10.8-1.7 1.5-1.1 1.5 7.2 1.5 15.4 12 22.4 12 22.4z"
                fill="#C6C2BF"
                stroke="none"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Sivilstand
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          Gift
           (
          12.12.2005
          )
        </p>
        <p
          className="typo-normal"
        >
          Sperret adresse, strengt fortrolig
        </p>
        <p
          className="typo-normal"
        >
          
           
           
        </p>
        <p
          className="typo-normal"
        >
          
        </p>
        <p
          className="typo-normal"
        >
          Bor ikke med bruker
        </p>
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              height={64}
              viewBox="0 0 64 64"
              width={64}
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M32 0a32 32 0 1 0 0 64 32 32 0 0 0 0-64zm0 8a3.9 3.9 0 1 1 0 7.7 3.9 3.9 0 0 1 0-7.8zm10 15.5v10.8c0 .9-.7 1.6-1.6 1.6-1 0-1.8-.7-1.8-1.6v-8.9c0-.4-.3-.8-.7-.8-.4 0-.7.3-.7.7v28.5a2.2 2.2 0 1 1-4.5 0v-18a.7.7 0 0 0-1.4 0v18a2.2 2.2 0 0 1-4.4 0V25.4a.7.7 0 0 0-1.5 0V34.3c0 .9-.8 1.6-1.8 1.6s-1.7-.7-1.7-1.6V23.5v-.7c0-3.4 3.6-5.8 7-6H35c3.4.2 7 2.6 7 6v.7z"
                fill="rgba(0, 0, 0, 0.5)"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Barn
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          Sperret adresse, fortrolig
        </p>
        <p
          className="typo-normal"
        >
          
           
           
        </p>
        <p
          className="typo-normal"
        >
          
        </p>
        <p
          className="typo-normal"
        />
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              height={64}
              viewBox="0 0 64 64"
              width={64}
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M32 0a32 32 0 1 0 0 64 32 32 0 0 0 0-64zm0 8a3.9 3.9 0 1 1 0 7.7 3.9 3.9 0 0 1 0-7.8zm10 15.5v10.8c0 .9-.7 1.6-1.6 1.6-1 0-1.8-.7-1.8-1.6v-8.9c0-.4-.3-.8-.7-.8-.4 0-.7.3-.7.7v28.5a2.2 2.2 0 1 1-4.5 0v-18a.7.7 0 0 0-1.4 0v18a2.2 2.2 0 0 1-4.4 0V25.4a.7.7 0 0 0-1.5 0V34.3c0 .9-.8 1.6-1.8 1.6s-1.7-.7-1.7-1.6V23.5v-.7c0-3.4 3.6-5.8 7-6H35c3.4.2 7 2.6 7 6v.7z"
                fill="rgba(0, 0, 0, 0.5)"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Barn
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          Sperret adresse, strengt fortrolig
        </p>
        <p
          className="typo-normal"
        >
          
           
           
        </p>
        <p
          className="typo-normal"
        >
          
        </p>
        <p
          className="typo-normal"
        />
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs>
                <mask
                  id="jentebarnmask"
                >
                  <circle
                    cx={10}
                    cy={10}
                    fill="#FFF"
                    r={10}
                  />
                  <path
                    d="M11.4 13.47v2.85c0 .32-.27.58-.59.58a.58.58 0 0 1-.58-.58v-2.85h-.4v2.85c0 .32-.25.58-.57.58a.58.58 0 0 1-.59-.58v-2.85H7.43l.69-3.76-.97 1.53a.56.56 0 1 1-.9-.66l1.6-2.5c.33-.45.5-.56 1.28-.56h1.82c.78 0 .95.1 1.27.55l1.61 2.51a.56.56 0 0 1-.42.93c-.2 0-.38-.11-.47-.27l-.98-1.53.69 3.76h-1.26zM12.1 5.31a2.07 2.07 0 0 1-4.13 0 1.35 1.35 0 0 1-1.92.76 1.35 1.35 0 0 1 1.96-1.29 2.07 2.07 0 0 1 4.05 0 1.35 1.35 0 0 1 1.95 1.29 1.35 1.35 0 0 1-1.91-.76z"
                    fill="#000"
                  />
                </mask>
              </defs>
              <rect
                fill="#C86151"
                height="20"
                mask="url(#jentebarnmask)"
                width="20"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Jente
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          Siri Bellomnavn Aremark
           
          (15)
           
        </p>
        <p
          className="typo-normal"
        >
          10117100000
        </p>
        <p
          className="typo-normal"
        >
          Bor med bruker
        </p>
      </div>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs>
                <mask
                  id="guttebarnmask"
                >
                  <circle
                    cx={10}
                    cy={10}
                    fill="#FFF"
                    r={10}
                  />
                  <path
                    d="M11.382 12.963v3.4a.58.58 0 0 1-.582.583.58.58 0 0 1-.583-.582v-3.4H9.83v3.4a.58.58 0 0 1-.582.582.58.58 0 0 1-.582-.582v-3.4h-.548V9.195l-.97 1.537a.558.558 0 1 1-.894-.661l1.606-2.509c.319-.446.498-.557 1.274-.557h1.861c.777 0 .956.111 1.274.557l1.607 2.509a.56.56 0 1 1-.894.661l-.97-1.537v3.767h-.629zM10.023 6.795a2.07 2.07 0 0 0 2.069-2.068c0-1.14-.928-2.07-2.069-2.07-1.14 0-2.068.93-2.068 2.07a2.07 2.07 0 0 0 2.068 2.068"
                    fill="#000"
                  />
                </mask>
              </defs>
              <rect
                fill="#3385D1"
                height="20"
                mask="url(#guttebarnmask)"
                width="20"
              />
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            Gutt
          </h4>
        </div>
        <p
          className="typo-normal"
        >
          Ola Ballomnavn Aremark
           
           
        </p>
        <p
          className="typo-normal"
        >
          10117100100
        </p>
        <p
          className="typo-normal"
        >
          Bor ikke med bruker
        </p>
      </div>
    </section>
    <section>
      <div
        className="c2"
      >
        
        <h3
          className="typo-undertittel"
        >
          NAV-kontor
        </h3>
      </div>
      <div
        className="c3"
      />
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          <div
            className="c5"
          >
            <svg
              className="prefix__navLogo"
              viewBox="0 0 269 169"
              width={90}
            >
              <g
                fill="none"
                fillRule="evenodd"
              >
                <path
                  d="M125.3 169a84.5 84.5 0 110-169 84.5 84.5 0 010 169zM0 121.3l17.3-42.8h16.6l-17.3 42.8zm213 0l17-42.8h9.1l-17 42.8z"
                  fill="#C30000"
                />
                <path
                  d="M.6 43.4L17.6.7h4.8l-17 42.7z"
                  fill="#C30000"
                  mask="url(#prefix__b)"
                  transform="translate(246 78)"
                />
                <path
                  d="M197.4 78.6h-15s-1 0-1.5 1l-8.3 25.4-8.3-25.5c-.3-.9-1.4-.9-1.4-.9h-28.8c-.7 0-1.2.6-1.2 1.2v8.6c0-6.8-7.3-9.8-11.6-9.8-9.5 0-16 6.3-18 15.9 0-6.3-.6-8.6-2.3-11a11 11 0 00-3.1-2.9c-2.6-1.4-4.9-2-9.8-2h-5.7s-1 0-1.5 1l-5.2 13V79.8c0-.6-.5-1.2-1.1-1.2H61.2s-1 0-1.4 1L54.3 93s-.5 1.3.7 1.3h5.2v25.8c0 .7.5 1.2 1.1 1.2h13.3c.6 0 1.1-.5 1.1-1.2V94.4h5.2c3 0 3.5.1 4.7.6.7.3 1.3.8 1.7 1.4.7 1.4.8 3 .8 7.6v16.2c0 .7.5 1.2 1.2 1.2H102s1.4 0 2-1.5l2.8-7c3.7 5.3 9.9 8.5 17.5 8.5h1.7s1.4 0 2-1.5l4.9-12v12.3c0 .7.5 1.2 1.2 1.2H147s1.4 0 2-1.5l5.2-13c.2-1-1.1-1-1.1-1h-4.7V84l14.6 36c.6 1.5 2 1.5 2 1.5h15.3s1.4 0 2-1.5l16.1-40c.6-1.4-1-1.4-1-1.4zM132.9 106h-8.7a6.3 6.3 0 110-12.5h2.4c3.5 0 6.3 2.8 6.3 6.2v6.3z"
                  fill="#FEFEFE"
                />
              </g>
            </svg>
          </div>
          <h4
            className="typo-element"
          >
            0118 NAV Aremark
          </h4>
        </div>
        <div>
          <br />
          <div
            className="c6"
          >
            <p
              className="typo-etikett-liten"
            >
              Besøksadresse
            </p>
          </div>
          <p
            className="typo-normal"
          >
            
            Rådhuset 6A
          </p>
          <p
            className="typo-normal"
          >
            1798 Aremark
          </p>
          <br />
          <div
            className="c6"
          >
            <p
              className="typo-etikett-liten"
            >
              Åpningstider
            </p>
          </div>
          <dl
            className="c8"
          >
            <dt>
              <p
                className="typo-normal"
              >
                <span
                  className="c9"
                >
                  mandag
                </span>
              </p>
            </dt>
            <dd>
              <p
                className="typo-normal"
              >
                09.00
                 
                –
                 
                14.30
              </p>
            </dd>
            <dt>
              <p
                className="typo-normal"
              >
                <span
                  className="c9"
                >
                  tirsdag
                </span>
              </p>
            </dt>
            <dd>
              <p
                className="typo-normal"
              >
                09.00
                 
                –
                 
                14.30
              </p>
            </dd>
            <dt>
              <p
                className="typo-normal"
              >
                <span
                  className="c9"
                >
                  onsdag
                </span>
              </p>
            </dt>
            <dd>
              <p
                className="typo-normal"
              >
                09.00
                 
                –
                 
                14.30
              </p>
            </dd>
            <dt>
              <p
                className="typo-normal"
              >
                <span
                  className="c9"
                >
                  torsdag
                </span>
              </p>
            </dt>
            <dd>
              <p
                className="typo-normal"
              >
                09.00
                 
                –
                 
                14.30
              </p>
            </dd>
            <dt>
              <p
                className="typo-normal"
              >
                <span
                  className="c9"
                >
                  fredag
                </span>
              </p>
            </dt>
            <dd>
              <p
                className="typo-normal"
              >
                09.00
                 
                –
                 
                14.30
              </p>
            </dd>
          </dl>
          <br />
        </div>
        <a
          className="lenke"
          href="https://norg2-frontend.nais.preprod.local/#/startsok?enhetNr=0118"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="typo-normal"
          >
            Mer informasjon om kontoret
          </span>
        </a>
      </div>
    </section>
    <section>
      <div
        className="c2"
      >
        <div
          className="c5"
        >
          <svg
            viewBox="0 0 100 100"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="50"
              cy="50"
              fill="#FFB249"
              r="50"
            />
            <line
              stroke="#3E3832"
              strokeLinecap="round"
              strokeWidth="4"
              x1="50"
              x2="50"
              y1="25"
              y2="57"
            />
            <circle
              cx="50"
              cy="72"
              fill="#3E3832"
              r="4.5"
            />
          </svg>
        </div>
        <h3
          className="typo-undertittel"
        >
          Tilrettelagt kommunikasjon
        </h3>
      </div>
      <div
        className="c3"
      />
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          
        </div>
        <p
          className="typo-normal"
        >
          Tolkehjelp
        </p>
      </div>
    </section>
    <section>
      <div
        className="c2"
      >
        <div
          className="c5"
        >
          <svg
            viewBox="0 0 100 100"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="50"
              cy="50"
              fill="#FFB249"
              r="50"
            />
            <line
              stroke="#3E3832"
              strokeLinecap="round"
              strokeWidth="4"
              x1="50"
              x2="50"
              y1="25"
              y2="57"
            />
            <circle
              cx="50"
              cy="72"
              fill="#3E3832"
              r="4.5"
            />
          </svg>
        </div>
        <h3
          className="typo-undertittel"
        >
          Bruker er under vergemål
        </h3>
      </div>
      <div
        className="c3"
      />
      <p
        className="typo-normal"
      >
        Enslig mindreårig asylsøker
      </p>
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          
          <h4
            className="typo-element"
          >
            Verge
          </h4>
        </div>
        <div
          className="c10"
        >
          <p
            className="typo-normal"
          >
             Solli Simen
          </p>
          <p
            className="typo-normal"
          >
            21042900076
          </p>
          <p
            className="typo-normal"
          >
            Advokat
          </p>
        </div>
        <p
          className="typo-element"
        >
          Mandat
        </p>
        <p
          className="typo-normal"
        >
          Ivareta personens interesser innenfor det personlige og økonomiske området herunder utlendingssaken (kun for EMA)
        </p>
        <p
          className="typo-normal"
        >
          Her kommen en fritekst som er sånn passelig lang
        </p>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Fylkesmannen i Østfold
            <br />
            27.03.2016 – 30.06.2013
          </p>
        </div>
      </div>
    </section>
    <section>
      <div
        className="c2"
      >
        <div
          className="c5"
        >
          <svg
            height={100}
            viewBox="0 0 100 100"
            width={100}
            xmlns="http://www.w3.org/2000/svg"
          >
            <defs>
              <mask
                id="sikkerhetstiltakmask"
              >
                <rect
                  fill="#fff"
                  height="100%"
                  width="100%"
                />
                <line
                  stroke="#000"
                  strokeLinecap="round"
                  strokeWidth="4"
                  x1="50"
                  x2="50"
                  y1="35"
                  y2="67"
                />
                <circle
                  cx="50"
                  cy="82"
                  fill="#000"
                  r="4.5"
                />
              </mask>
            </defs>
            <polygon
              fill="#BA3A26"
              mask="url(#sikkerhetstiltakmask)"
              points="50,5 95,95 5,95"
              stroke="#BA3A26"
              strokeLinejoin="round"
              strokeWidth="4"
            />
          </svg>
        </div>
        <h3
          className="typo-undertittel"
        >
          Sikkerhetstiltak
        </h3>
      </div>
      <div
        className="c3"
      />
      <div
        className="c4"
      >
        <div
          className="c2"
        >
          
        </div>
        <div
          className="c6"
        >
          <p
            className="typo-etikett-liten"
          >
            Gyldig: 
            12.12.2005
             - 
            12.12.2025
          </p>
        </div>
        <p
          className="typo-normal"
        >
          Får verken møte opp på NAV-kontor eller ringe på telefon grunnet truende oppførsel og språk.
        </p>
      </div>
    </section>
    <div
      className="c11"
    >
      <a
        className="lenke"
        href="/modiabrukerdialog/person/10108000398#!brukerprofil"
      >
        <span
          className="typo-normal"
        >
          Administrer brukerprofil
        </span>
      </a>
    </div>
  </div>
</section>
`;
